webpack.config.js

```diff
@@ -0,0 +1,27 @@
+const path = require('path');
```

Comment:
Could you add a blank line after this line to separate builtin imports from node_modules imports?